azurerm/resource_arm_virtual_machine_scale_set.go

```diff
@@ -67,6 +67,47 @@ func resourceArmVirtualMachineScaleSet() *schema.Resource {
 				Required: true,
 			},
 
+			"automatic_os_upgrade": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+
+			"rolling_upgrade_policy": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"max_batch_instance_percent": {
+							Type:     schema.TypeInt,
+							Optional: true,
+							Default:  20,
+						},
+
+						"max_unhealthy_instance_percent": {
+							Type:     schema.TypeInt,
+							Optional: true,
+							Default:  20,
+						},
+
+						"max_unhealthy_upgraded_instance_percent": {
+							Type:     schema.TypeInt,
+							Optional: true,
+							Default:  5,
+						},
+
+						"pause_time_between_batches": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Default:  "PT0S",
```

Comment:
are these 4 defaults the default set by Azure?

Reply:
I believe they are, yes.

Reply:
Yes it is.